### PR TITLE
Timer UI

### PR DESCRIPTION
**Pull request by LiChung-Hao**
Timer in minutes
Menu UI
Credit Scene
Time up scene

### CODE REVIEW

File: Assets/Scripts/Content.cs [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 using UnityEngine.SceneManagement;
+using System.Security.Cryptography.X509Certificates;
 
 public class Content : MonoBehaviour
 {
     public TextMeshProUGUI m_textComponent; //reference tmpro to text component
     const int TALKERNUM = 2;
     const int LINESNUM = 9;
-    private string[] m_lines=new string[LINESNUM];
+    private string[] m_lines = new string[LINESNUM];
```
Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:13]: Just remember in any place with m_ or s_ the next word needs to be capital so in this case m_Lines

File: Assets/Scripts/Content.cs [hunk 2]
```diff
@@ -81,7 +82,7 @@ void NextLine()
 
     public void SentenceBehaviour()
     {
-        if (Input.GetMouseButtonDown(0) || Input.GetButtonDown("Add1"))
+        if (Input.GetMouseButtonDown(0))
```
Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:85]: You'll need to add Input.GetButtonDown("Add1") back into the if statement for the controller to work

File: Assets/Scripts/Content.cs [hunk 3]
```diff
@@ -111,7 +112,7 @@ public void SentenceBehaviour()
             m_talkerIndex = 1;
         }
 
-        if (m_conversationOver || Input.GetKeyDown(KeyCode.Escape) || Input.GetButtonDown("Pause1"))
+        if (m_conversationOver || Input.GetKeyDown(KeyCode.Escape))
```
Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:115]: Same thing here. Needs to be added back

Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:115]: m_ConversationOver

File: Assets/Scripts/Content.cs [hunk 4]
```diff
@@ -121,9 +122,9 @@ public void SentenceBehaviour()
         //Make the character in the scene who is not talking stop the mouth talking animation
         for (int i = 0; i < TALKERNUM; i++)
         {
-            if (i != m_talkerIndex || m_textComponent.text == m_lines[m_index]) 
+            if (i != m_talkerIndex || m_textComponent.text == m_lines[m_index])
```
Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:125]: m_TalkerIndex, m_Index

File: Assets/Scripts/Content.cs [hunk 5]
```diff
@@ -133,13 +134,18 @@ public void SentenceBehaviour()
     //These 4 functions below are for talkers' and their mouths' animation
     public void StartTalker(int m_talkerIndex)
     {
-        StartTalking(m_storyTellerMouth[m_talkerIndex]);
         m_storyTeller[m_talkerIndex].SetActive(true);
+
+        StartTalking(m_storyTellerMouth[m_talkerIndex]);
```
Comment (JoshuaDaSilva) [inline Assets/Scripts/Content.cs:139]: m_StoryTeller, m_StoryTellerMouth

**Review by JoshuaDaSilva — CHANGES_REQUESTED** [review]
Overall, great work Li. I'd also change the timer to at least a minute because it is only at 2 seconds right now. I'm sure it was just for testing purposes but I would still change it prior to merging so that we don't have to worry about changing it in another branch.
Keep up to great work Li.

**Review by JoshuaDaSilva — APPROVED** [review]
Great work Li. The changes to the timing will help the game going forward

File: Assets/Sprites/Credit/snowedinlogo.png
Comment (Dunwaru) [inline Assets/Sprites/Credit/snowedinlogo.png:1]: Dom we have permission of this?

File: Assets/Sprites/SceneBG/BG1-MENU.png
Comment (Dunwaru) [inline Assets/Sprites/SceneBG/BG1-MENU.png:1]: I like this image

File: Assets/Sprites/Credit/Ethan.png
[the remark this answers is not in the document]
Comment (LiChung-Hao) [reply Assets/Sprites/Credit/Ethan.png:1]: Sure

File: Assets/Sprites/Credit/snowedinlogo.png
Comment (LiChung-Hao) [reply Assets/Sprites/Credit/snowedinlogo.png:1]: remove it first, wait for Johanne's (SnowedIn HR)feedback

**Review by Dunwaru — APPROVED** [review]

**Pull request merged**